pkgs/development/python-modules/hatch-nodejs-version/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, hatchling
+}:
+
+buildPythonPackage rec {
+  pname = "hatch-nodejs-version";
+  version = "0.3.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "agoose77";
+    repo = "hatch-nodejs-version";
+    rev = "v${version}";
+    sha256 = "sha256-TBvqXjka8poQ8xxQ/H1hFYRLqnP1024uan1d9b95ags=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+  ];
+
+  propagatedBuildInputs = [
+    hatchling
+  ];
```

should only need one of these, assuming it's a python package
```suggestion
  propagatedBuildInputs = [
    hatchling
  ];
```